### PR TITLE
Added landing, docs pages live - https://beadsui.netlify.app

### PR DESCRIPTION
**Pull request by Developers-codz**

### CODE REVIEW

**Review by nayyyhaa — APPROVED** [review]
Clean UI. Nice work Kajal! 👍

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks great!!! Just fix the indentation. 👍🏽

**Comment by Developers-codz** [comment]
>

**Comment by Developers-codz** [comment]
> 

thank you Neha :)

**Comment by Developers-codz** [comment]
> 

yep working on that. Thank you :)

File: docsite/docs.css [hunk 1]
```diff
@@ -48,10 +48,6 @@ aside>h1::after{
     bottom: 5px;
     display: inline-block;
     
-}
-aside > ul > li > a {
-    
-
 }
 aside > ul > li > a:hover{
```
Comment (yash9263) [inline docsite/docs.css]: can give `a` tag class like `aside-link` or `nav-link` instead of using combinators

**Comment by yash9263** [comment]
looks good to me

**Review by b0llu — APPROVED** [review]
Looks good to me!

File: docsite/docs.css (hunk 1 above)
Comment (Developers-codz) [reply docsite/docs.css]: Done! Thanks for reviewing :)

**Pull request merged**